CMakeLists.txt

```diff
@@ -387,6 +428,16 @@ target_include_directories(${BIN_TARGET} PRIVATE
   3rdParty/Radon/Radon/include
   3rdParty/libsmacker)
 
+if(DISCORD_INTEGRATION AND WIN32)
+  target_include_directories(${BIN_TARGET} PRIVATE "${discordsrc_SOURCE_DIR}/..")
+
+  set(DISCORD_LIB_DIR "${discordsrc_SOURCE_DIR}/lib/x86_64")
+
+  target_link_libraries(discord PRIVATE "${DISCORD_LIB_DIR}/discord_game_sdk.dll.lib")
+  target_link_libraries(${BIN_TARGET} PRIVATE discord)
+  file(COPY "${DISCORD_LIB_DIR}/discord_game_sdk.dll" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")

```

Comment:
Replace `dll` with `${CMAKE_SHARED_LIBRARY_SUFFIX}` so that this also works for Mac and Linux